### ISSUE TITLE
gltfio: Create multiple renderable instances of FilamentAsset with shared buffers

### ISSUE THREAD

**Issue by FabianRueckert** [google/filament#2344]
**Is your feature request related to a problem? Please describe.**

I would like to load a GLTF object from file only once and then create multiple renderable instances of that object.

However, the functions
```
FilamentAsset createAssetFromBinary(Buffer buffer)
FilamentAsset createAssetFromJson(Buffer buffer)
```
from AssetLoader parse a GLTF buffer and directly create an instance of a hierarchy of filament renderable entities.

```
ResourceLoader.loadResources(FilamentAsset asset)
```
loads the corresponding textures and buffers to the GPU.

With this API, for every object instance that I want to render, I would have to parse the GLTF file again and vertex buffers and textures would be duplicated.

**Describe the solution you'd like**

I would like to have an API that allows me to separate GLTF loading from instance creation.
Maybe something like:
```
RenderableData data = assetLoader.loadAssetDataFromBinary(gltfBuffer);
resourceLoader.loadResources(data);

FilamentAsset asset = data.createInstance();
```

**Comment by romainguy** [comment]
It would make sense to add `createAssetFromAsset(FilamentAsset)` or something equivalent.

**Comment by prideout** [comment]
Duplicate of #1513.
